### ISSUE TITLE
WSG: Tab sequence not fully functioning 

### ISSUE THREAD

**Issue by chelkahn** [ecsion/eSeaGrant-Client-Bug-Tracker#686]
Tab order for two of the forms is not functioning properly - see 2 screenshots. This seems to be linked to the previous "issue" relating to missing "<Previous" and "Next>" buttons.

![screen shot 2017-04-25 at 9 29 17 am](https://cloud.githubusercontent.com/assets/26942420/25396612/ffcb8300-2999-11e7-914a-b4f314ecdede.png)
![screen shot 2017-04-25 at 9 29 09 am](https://cloud.githubusercontent.com/assets/26942420/25396613/ffccd0a2-2999-11e7-8517-e160f38da30e.png)

**Comment by ecsion** [comment]
Chelsea, Could you please add a URL

**Comment by ecsion** [comment]
We will fix this issue in the next couple of days. Meanwhile as a work around, please make sure there are no extraneous space after the form name. So for example "Full Proposal Project Narrative" is correct but this issue will happen if you named the form "Full Proposal Project Narrative "  (notice the extra space after name. We corrected couple of forms for you. If this issue is still happening, check the form name and remove the extra space after the name.

**Comment by chelkahn** [comment]
fabulous! this work around will do just fine for now - everything is working well. Thank you!
